apps/content/services/recitation.py

```
@@ -2,30 +2,137 @@
 
-        return self.repo.list_recitations_qs(
-            publisher_q, filters_dict=filters_dict, annotate_surahs_count=annotate_surahs_count
+    def create_recitation(
+        self,
+        *,
+        publisher_id: int,
+        name_ar: str,
+        name_en: str,
+        description_ar: str,
+        description_en: str,
+        license: str,
```

Ruff (0.15.9)

[error] Function argument `license` is shadowing a Python builtin

(A002)